lms/djangoapps/course_api/README.rst

```diff
@@ -0,0 +1,17 @@
+Status: Maintenance
+
+Responsibilities
+================
+The course_api app provides a read-only REST endpoint that allows you to quickly query and filter for certain block types across an entire course run's courseware content. For instance, you might want a list of all videos in the course, or a list of all problems in the course. The container/child relations are also returned.
```

Comment:
Do you want to distinguish between the Course Blocks API and the Courses API?  The former is described here.  The latter is used to get information on the Courses in the system.